vendor/github.com/diktyo-io/appgroup-api/pkg/apis/appgroup/v1alpha1/types.go

```diff
@@ -27,6 +27,7 @@ const (
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +kubebuilder:resource:scope=Namespaced,shortName=ag
+// +kubebuilder:metadata:annotations="api-approved.kubernetes.io=https://github.com/kubernetes-sigs/scheduler-plugins/pull/432"
```

Comment:
Let's try to avoid modify content in vendor/. Instead, we can remove `api-approved.kubernetes.io` in the CRD yamls that don't opt-in `+kubebuilder:metadata:annotations` in their types.go.